### DISCUSSION TITLE
Scroll after go to next form step

### DISCUSSION THREAD

**Discussion by linkvn** [Budibase/budibase#10827] [Help] [answered]
I have a multiple steps form and i want to scroll to top of page after goto next form step. But i dont know how to, please help!

**Comment by ConorWebb96** [comment]
Hey @linkvn,

I've tried a few workarounds to get this to work within Budibase none have worked. Due to this I would suggest writing up an enhancement request for the form step action. All that would likely be needed is a checkbox on the form step action, if check it would scroll the user back to the top of the form whenever any of the step actions are used.

Hopefully this makes sense. 
I'll look for a bit longer for a potential workaround, but I do think this will require an enhancement to the core functionality.

**Reply by linkvn** [reply]
@ConorWebb96 the same with me. we should add it into core functions. Let me know if anyone find workaround solution :)

**Reply by melohagan** [reply, marked as answer]
This has been discussed before, open feature request here! https://github.com/Budibase/budibase/issues/3671
